src/Component/Best.jsx

```diff
@@ -1,14 +1,65 @@
-import React, {Component} from "react";
-
+import React, { Component } from "react";
+import axios from "axios";
+import moment from "moment";
+import CONSTANTS from "../Services/Hnapi";
+const { baseUrl } = CONSTANTS;
 export default class Best extends Component {
-    render() {
-        return (
+  constructor() {
+    super();
+  }
+  state = {
+    getBestItem: [],
+  };
+
+  componentDidMount() {
+    const promises = [];
+    axios
```

Comment:
for using axios, 

create a http.js file inside utils folder.

![image](https://user-images.githubusercontent.com/22271897/121465570-c24d6080-c9d5-11eb-9a7b-4c2cffd8dfa0.png)

keep this in http.js file,
```suggestion
import axios from "axios";

import configs from "../config";

const http = axios.create({
  headers: {
    "Content-Type": "application/json"
  },
  baseURL: configs.baseURI
});

export default http;```

Reply:
Now perform axios call inside service folder

```suggestion
    import config from "../config";
import http from "../utils/http";

const { api } = config;

/**
 * Function to create a comment
 * 
 * @param data any
 * @param id string
 */
export async function createNewComment(data: any, id: string) {
  const url = `${api.endpoints.posts + "/" + id + api.endpoints.comments}`;
  const response = await http.post(url, data);
 
  return response;
}
```

![image](https://user-images.githubusercontent.com/22271897/121465746-e8730080-c9d5-11eb-95f1-c95de27e60ed.png)

Reply:
and use it like this:
![image](https://user-images.githubusercontent.com/22271897/121466060-7a7b0900-c9d6-11eb-8988-3e7171b7f332.png)

Reply:
config file :

![Uploading image.png…]()